src/vs/code/electron-main/windows.ts

```diff
@@ -602,6 +605,18 @@ export class WindowsManager implements IWindowsService {
 			iPathsToOpen = this.cliToPaths(openConfig.cli, ignoreFileNotFound);
 		}
 
+		// Add any existing backup workspaces
+		if (openConfig.restoreBackups) {
+			// TODO: Ensure the workspaces being added actually have backups
+			this.backupService.getBackupWorkspaces().forEach(ws => {
```

Comment:
Provided I understand correctly, does opening the same folder twice restore backups? This should never happen as `openConfig.restoreBackups` is only ever true on the first window launch. https://github.com/Microsoft/vscode/pull/12400/files#diff-d8dfcbab1cae48b2cbc3cabc682c9cd4R258